lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -70,6 +81,7 @@ class RenderBlockingResources extends Audit {
 
     const metricSettings = {throttlingMethod: 'simulate'};
     const metricComputationData = {trace, devtoolsLog, simulator, settings: metricSettings};
+    // @ts-ignore - TODO(bckenny): allow optional `throttling` settings
```

Comment:
@patrickhulce maybe we want a new type for settings required for metrics rather than the full config settings